internal/credential/store.go

```diff
@@ -16,114 +16,25 @@ limitations under the License.
 package credential
 
 import (
-	"context"
-	"fmt"
-	"os"
-
-	"github.com/docker/cli/cli/config"
-	"github.com/docker/cli/cli/config/configfile"
-	"github.com/docker/cli/cli/config/credentials"
-	"github.com/docker/cli/cli/config/types"
-	"oras.land/oras-go/v2/registry/remote/auth"
+	credentials "github.com/oras-project/oras-credentials-go"
 )
 
-// Store provides credential CRUD operations.
-type Store struct {
-	configs []*configfile.ConfigFile
-}
-
-// NewStore generates a store based on the passed in config file path.
-func NewStore(configPaths ...string) (*Store, error) {
+// NewStore generates a store based on the passed-in config file paths.
+func NewStore(configPaths ...string) (store credentials.Store, err error) {
```

Comment:
Is there any particular reason to use _named_ parameters? Please avoid them unless necessary.